cloud/aws/services/ec2/tags.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ec2
```

Comment:
This file is pretty much a copy of https://github.com/kubernetes/kubernetes/blob/e4b0a935fa393944b6322fa6ef0970d858ad70f6/pkg/cloudprovider/providers/aws/tags.go for now.

Reply:
Not sure if we should re-use the same tags or use something namespaced under `sigs.k8s.io`.

Reply:
I still need to finish the document that I started here: https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/66/files, but I think we should re-use the same tags as existing tooling, but we should also add a specific aws-provider tag, say `sigs.k8s.io/cluster-api-provider-aws=true` in the case where `kubernetes.io/cluster/<name>=owned` to be able to differentiate between components we own vs components that are owned by other controllers/tooling.